pyfolio/timeseries.py

```diff
@@ -410,23 +312,19 @@ def alpha_beta(returns, factor_returns):
     float
         Beta.
 
-"""
-
-    ret_index = returns.index
-    beta, alpha = sp.stats.linregress(factor_returns.loc[ret_index].values,
-                                      returns.values)[:2]
+    """
 
-    return alpha * APPROX_BDAYS_PER_YEAR, beta
+    return qrisk.alpha_beta(returns, factor_returns=factor_returns)
 
 
 def alpha(returns, factor_returns):
-    """Calculates annualized alpha.
+    """Calculates alpha.
```

Comment:
It previously assumed daily returns and annualized alpha. It's unclear to me what it should be.

@Jstauth @justinlent Is it conventional to annualize alpha?